packages/nimble-components/src/table/components/group-row/index.ts

```diff
@@ -66,6 +66,12 @@ export class TableGroupRow extends FoundationElement {
     @observable
     public animationClass = '';
 
+    /**
+     * @internal
+     */
+    @attr({ attribute: 'allow-hover-styling', mode: 'boolean' })
```

Comment:
nit: Including `styling` in the name feels weird / too leaky about the implementation. Can we do `allow-hover`?